mora/service/manager.py

```diff
@@ -119,22 +120,21 @@ def edit_manager(employee_uuid, req):
     if keys.MANAGER_TYPE in data:
         update_fields.append((
             mapping.ORG_FUNK_TYPE_FIELD,
-            {'uuid': data.get(keys.MANAGER_TYPE).get('uuid')},
+            {'uuid': common.get_mapping_uuid(data, keys.MANAGER_TYPE)},
         ))
 
     if keys.ORG_UNIT in data:
-        org_unit_uuid = data.get(keys.ORG_UNIT).get('uuid')
         update_fields.append((
             mapping.ASSOCIATED_ORG_UNIT_FIELD,
-            {'uuid': org_unit_uuid},
+            {'uuid': common.get_mapping_uuid(data, keys.ORG_UNIT)},
         ))
 
-    if keys.RESPONSIBILITY in data:
+    for responsibility in common.checked_get(data, keys.RESPONSIBILITY, []):
```

Comment:
I've been thinking about this a bit, and I just can't see why we'd need a new mapping. Apart from allowing more than one values, the behaviour is exactly the same as previously. Sure, we could add a bit of checking to ensure that we don't _accidentally_ write multiple values to a relation, but other than that, nothing has changed apart from a slight generalisation.

The previous behaviour of `ADAPTED_ZERO_TO_MANY` was:

> When I give you this one value, overwrite anything during that time with this new value. Obviously, the value only affects one time interval.

The new behaviour is:

> When I give you these new values, overwrite anything during that time with this new value. The values may only affect one time interval.

In fact, one might even argue that the semantics haven't changed at all, but rather generalised to make a ‘value’ to also mean something represented as two (or more) distinct relations.

Reply:
But the behaviour _isn't_ exactly the same as previously. We can now write both single values, or multiple values to this field, but both of these cases clearly need to be handled separately. 

We have no way of determining whether we're supposed to expect one value or several, just from the mapping, which is why we're now resort to having specific cases for `responsibility` rather than just letting it mesh with the existing logic.